arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -83,22 +108,40 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           }
         ],        
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. Returns a list of all of the Media Services accounts in a resource group.",
             "schema": {
               "$ref": "#/definitions/MediaServiceCollection"
+            },
+            "examples": {

```

Comment:
key off the examples on MIME type in the responses section. The example for a model in the definitions section is free form.